Makefile

```diff
@@ -246,10 +242,16 @@ KUSTOMIZE=$(shell which kustomize)
 endif
 
 # Generate bundle manifests and metadata, then validate generated files.
-bundle: manifests
+bundle: manifests kustomize
```

Comment:
can you add this to the make help with the optional/required args?

Reply:
added